### PR TITLE
Update chocolatey package to tektoncd-cli version 0.7.0

### PR DESCRIPTION
**Pull request by sootysec**

### CODE REVIEW

**Review by danielhelfand — CHANGES_REQUESTED** [review]
@ducksecops Thanks so much for this! 

Unfortunately, there was an issue releasing `v0.7.0` that is requiring a minor release today. There was a [`gotest.tools` issue](https://github.com/gotestyourself/gotest.tools/pull/184) that required us to use a fork of `gotest.tools` for the release.

If you would be willing, it would be great if we could repurpose this pr for the minor release (`v0.7.1`) today. Let me know, and I can update you when the release is available.

**Comment by danielhelfand** [comment]
Released for [v0.7.1](https://github.com/tektoncd/cli/releases/tag/v0.7.1).

**Comment by sootysec** [comment]
Hi,

Update to version 0.7.1.

Regards

File: tektoncd-cli/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-$url64       = 'https://github.com/tektoncd/cli/releases/download/v0.6.0/tkn_0.6.0_Windows_x86_64.zip' 
+$url64       = 'https://github.com/tektoncd/cli/releases/download/v0.7.1/tkn_0.7.1_Windows_x86_64.zip' 
 
 $packageArgs = @{
   packageName    = $packageName
   unzipLocation  = $toolsDir
   url64bit       = $url64
   softwareName   = 'tektoncd-cli*' 
-  checksum64     = '5d69844fbb2ddaeece6cb872c19c67805d96b19293d2473e2c60fa8886fb7fc7'
+  checksum64     = 'SHA256: 592C79C96FCE12B8E8DC4F853C1D8D6FAC76E18AC7A3B2118D44A238A5274875
```
Comment (danielhelfand) [inline tektoncd-cli/tools/chocolateyinstall.ps1]: Could you change to:

```suggestion
  checksum64     = '592c79c96fce12b8e8dc4f853c1d8d6fac76e18ac7a3b2118d44a238a5274875'
```

**Review by danielhelfand — COMMENTED** [review]
Just minor change, and could you also please squash this into one commit?

**Comment by danielhelfand** [comment]
@ducksecops Thanks! Could you please squash these commits into a single one?

**Comment by danielhelfand** [comment]
Closing this in favor of [33bfcb9](https://github.com/danielhelfand/chocolatey-tektoncd-cli/commit/33bfcb9d9e13b051ef41155985f7140162840b32).

**Pull request closed without merging**